decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -93,23 +132,53 @@ const AddCommentFormWithMutation = graphql(gql`
         __typename: 'Mutation',
         addComment: {
           __typename: 'Comment',
-          id: random.uuid(),
-          createdAt: (new Date()).toString(),
+          id: uuid(),
+          createdAt: moment().format("YYYY-MM-DD HH:mm:ss z"),
```

Comment:
Please localize the format. Check "Localized formats" in moment's documentation: http://momentjs.com/docs/#/displaying/format/